setup.py

```diff
@@ -64,6 +64,10 @@ def run_tests(self):
     version=__version__,
     packages=find_packages(),
 
+    data_files=[
```

Comment:
@huntc @fsat Does adding the binary file with `data_files` work for the native and pip3 distribution?